### PR TITLE
Kills off /obj/item/device

### PR DESCRIPTION
**Pull request by Fox-McCloud**
`/obj/item/device` has even less reason to exist than `/obj/item/weapon` does.

With weapon, at least it was defined somewhere and existed to set `hitsound`. 

device serves no purpose at all, either; it's just a case of more path bloat.

If this is acceptable, I'd like to humbly request this be merged as soon as possible; I really don't want to solve conflicts 20 different times.

### CODE REVIEW

File: tools/Runtime Condenser/Output.txt [hunk 1]
```diff
@@ -281,7 +281,7 @@ proc name: get fuel (/obj/item/weapon/weldingtool/proc/get_fuel)
 
 /obj/item/weapon/tank/internals/oxygen - 3 time(s).
 
-/obj/item/device/assembly/signaler - 3 time(s).
```
Comment (vuonojenmustaturska) [inline tools/Runtime Condenser/Output.txt]: this file doesn't need to be touched

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

File: tools/Runtime Condenser/Output.txt (hunk 1 above)
Comment (Fox-McCloud) [reply tools/Runtime Condenser/Output.txt]: Yep, whoops.

**Comment by Fox-McCloud** [comment]
@vuonojenmustaturska Addressed the issue.

File: code/game/objects/items/devices/chameleonproj.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/obj/item/device/chameleon
+/obj/item/chameleon
 	name = "chameleon-projector"
 	icon_state = "shield0"
 	flags_1 = CONDUCT_1 | NOBLUDGEON_1
```
Comment (AnturK) [inline code/game/objects/items/devices/chameleonproj.dm]: You didn't restore icon for things using the default device.dmi

**Review by AnturK — CHANGES_REQUESTED** [review]

**Comment by Rinoka** [comment]
also begging for a speedmerge, since i wanted to do a quick edit to radio code

File: code/game/objects/items/devices/chameleonproj.dm (hunk 2 above)
Comment (Fox-McCloud) [reply code/game/objects/items/devices/chameleonproj.dm]: I could just move everything in `device.dmi` to `items_and_weapons` as a catch all; finding all instance of defaulting to `device.dmi` will be extremely difficult.

Still, lemme know what you want me to do.

**Comment by Fox-McCloud** [comment]
@AnturK 

Attempted to go through and set the various devices to use `device.dmi` where I could, but I can't guarantee I caught them all.

If necessary, I can move all device icons to `items_and_weapons.dmi`

**Comment by ninjanomnom** [comment]
We want to move away from massed up icon files, not group them up even more

**Review by ninjanomnom — DISMISSED** [review]
Shoving all the icons together into one file is the wrong way to handle the problem

**Comment by Fox-McCloud** [comment]
So what changes do you have to request then? As my last post made clear, I *didn't* mash them all together and attempted to go through and add the icon path to them, so--is there something else, in particular that needs to be done?

**Comment by ninjanomnom** [comment]
The icon bot says you added a ton of icons to items_and_weapons.dmi, is that incorrect?

**Comment by Fox-McCloud** [comment]
Yes, I did, by mistake, and they were promptly removed the *very* next commit, if you look.

**Comment by ninjanomnom** [comment]
Ah ok, I guess the bot should be modified to show that

@lzimann

**Review by vuonojenmustaturska — APPROVED** [review]

**Pull request merged**